### ISSUE TITLE
"Unabridged API: unexpected kind 'page' (IGNORED)" when using @todo

### ISSUE THREAD

**Issue by LizardM4** [svenevs/exhale#88]
It looks like using `@todo` in Doxygen causes this warning to pop up during build:

```
(!) Unabridged API: unexpected kind 'page' (IGNORED)
```
The *todo* entry is then not visible anywhere in the final documentation.

How to reproduce
---
```
    /**
     * @todo This TODO directive causes <tt>(!) Unabridged API: unexpected kind 'page' (IGNORED)</tt>
     */
    class incomplete_class {
    };
```
I have a [separate project](https://github.com/LizardM4/respiration_issues) where I collected [this](https://github.com/LizardM4/respiration_issues/blob/master/src/todo.h) and other issues (still looking to dispatch them to the right repository); you can also check that out to reproduce.

Doxygen HTML output
---
![image](https://user-images.githubusercontent.com/3787759/80310834-fe772f00-87dc-11ea-8591-bdc309770a7c.png)

Exhale HTML output
---
![image](https://user-images.githubusercontent.com/3787759/80310839-0931c400-87dd-11ea-86a6-7394c74fa243.png)

**Comment by svenevs** [comment]
Well, that's certainly unexpected!  Hmmmm.  I will have to look into this `page` business, thanks for creating a repro example!  I don't know how easy this one will be to fix, in the meantime I wonder if you can get away with just using the [sphinx todo directive](https://www.sphinx-doc.org/en/master/usage/extensions/todo.html)?

- in `conf.py`'s `extensions` list add `"sphinx.ext.todo"` (same list where `"breathe"` and `"exhale"` are
- in docs, you have to do that in an `\rst` block:

    ```
    /**
     * \rst
     * .. todo:: implement me!
     * \endrst
     */
    ```
- somewhere in your docs, e.g. `docs/todo.rst` include the `.. todolist::` directive to dump them all out

Doesn't fix the actual problem here, but I think it could be a usable workaround?

**Comment by LizardM4** [comment]
Thanks! The todo list is more of "nice to have" for my project, but it definitely can be useful.
The workaround works perfectly, but I had to
 * add `todo_include_todos = True` to `conf.py`

Now that's what I'm getting:

![image](https://user-images.githubusercontent.com/3787759/80540060-df2cfd00-89a8-11ea-9b0d-72931417c129.png)

**Comment by schmoelder** [comment]
Hey, is there any update on this issue?

**Comment by svenevs** [comment]
as far as i know this can be closed, doxygen page support was added in #114 so i think it will work with regular old `\todo` in doxygen as well as the rst solution and `.. todolist::` is another viable contendor

please speak up if it's not fully solved

**Comment by brub43** [comment]
this also seems to happen with the `@deprecated` doxygen command.
